core/modulesource.go

```diff
@@ -350,6 +350,7 @@ func (src *ModuleSource) LoadContext(
 				Args: []dagql.NamedInput{
 					{Name: "path", Value: dagql.String(path)},
 					{Name: "exclude", Value: dagql.ArrayInput[dagql.String](dagql.NewStringArray(ignore...))},
+					{Name: "noCache", Value: dagql.NewBoolean(true)},
```

Comment:
worth adding a test for this behavior existing now

Reply:
oh yeah, agreed, for some reason i thought i already had but it's not in the diff

Reply:
i did already have it, but committed it as part of a refactor that i abandoned and squashed out. recovered. it passes.